integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -32,22 +32,23 @@ This enhanced support gives you greater flexibility and choice in managing your
 
 You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
 
-* {product-title-short} Scanner (recommended)
+* {product-title-short} Scanner V4: Beginning with release 4.4, a new scanner is introduced that is built on the {product-title-short} StackRox and Clair scanners. The new scanner, Scanner V4, supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. You must use this scanner in conjunction with the StackRox Scanner. See the installation documentation for more information.
+* StackRox Scanner: This scanner is the default scanner in {product-title-short}. It originates from a fork of the Clair v2 open source scanner. You must continue using this scanner for RHCOS node scanning and for active vulnerability management until that functionality is supported using Scanner V4 in a future release.
```

Comment:
> You must continue using this scanner for RHCOS node scanning and for active vulnerability management until that functionality is supported using Scanner V4 in a future release.

It's not just RHCOS nodes, but rather other kinds, too. RHCOS is just the main one and has better support. Let's also replace active VM with platform vulnerabilities (OpenShift, Kubernetes, Istio). We (I) do NOT want to add active VM support to Scanner V4, so let's not mention it